CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Eric Amorde](https://github.com/amorde)
   [#7093](https://github.com/CocoaPods/CocoaPods/pull/7093)
 
+* Prohibit SDK providers to use non-encrypted HTTP links for SDKs  
```

Comment:
s/to use/from using/

Reply:
Would also soften the wording of "Prohibit" to "Warn against" or something like that, but not a big deal.